conex/helpers/transforms/masks.py

```diff
@@ -33,7 +35,20 @@ def __call__(self, img):
         )
         for index, ij in enumerate(product(range(self.m), range(self.n))):
             i, j = ij
-            result.append(TF.erase(img, i * h_grid, j * w_grid, h_grid, w_grid, v=0))
+            h_cor = i * h_grid + self.gap[0]
```

Comment:
deconstruct the gap, and use local variables

```py
gap_left, gap_bottom, gap_up, gap_right = self.gap
```

or define constants

```py
class Direction:
   LEFT = 0
   BOTTOM = 1
   UP = 2
   RIGHT = 3

h_cor = i * h_grid + self.gap[Direction.LEFT]

```

Reply:
defining a class is overkill, deconstructing is pretty reasonable that a new commit should've elected this style.